distribution/tools/server-cli/src/main/java/org/elasticsearch/server/cli/SystemJvmOptions.java

```diff
@@ -12,12 +12,13 @@
 import org.elasticsearch.common.util.concurrent.EsExecutors;
 
 import java.util.List;
+import java.util.Map;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
 final class SystemJvmOptions {
 
-    static List<String> systemJvmOptions(Settings nodeSettings) {
+    static List<String> systemJvmOptions(Settings nodeSettings, final Map<String, String> sysprops) {
```

Comment:
Maybe call the variable `cliSysprops` to make it clear these were passed by the cli, they are not generally for the jvm we intend to create.